pkgs/by-name/mi/mixxc/package.nix

```diff
@@ -0,0 +1,65 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+  pkg-config,
+  libpulseaudio,
+  gtk4-layer-shell,
+  gtk4,
+  wrapGAppsHook4,
+  libxcb,
+  installShellFiles,
+  enableWayland ? true,
+  enableSass ? true,
+  enableX11 ? true,
+}:
+with lib;
```

Comment:
DON'T USE THIS!

Reply:
`with lib` should not be used there? I did not know that, sorry

Reply:
See https://github.com/NixOS/nixpkgs/issues/208242

Reply:
Oh, I see, is there anywhere one can see or learn of all of these little details that I've missed?
 It would be easier if I could avoid these things to begin with to prevent having dozens of commits

Reply:
> Oh, I see, is there anywhere one can see or learn of all of these little details that I've missed? It would be easier if I could avoid these things to begin with to prevent having dozens of commits

https://nix.dev/guides/best-practices.html